src/test/create-user.test.ts

```diff
@@ -0,0 +1,112 @@
+import axios from "axios";
+import { expect } from "chai";
+import { dataSource } from "../data-source";
+import { User } from "../entity/User";
+import * as bcrypt from "bcrypt";
+
+describe("Mutation createUser", () => {
+  afterEach(async function () {
+    await dataSource.query('TRUNCATE TABLE "user"');
+  });
+
+  const mutation = `mutation CreateUser($data: UserInput!) {
+        createUser(data: $data) {
+          id
+          email
+          name
+          birthDate
+        }
+      }
+      `;
+  const urlDB = "http://localhost:4000/graphql";
+  let input = {
```

Comment:
dentro do describe eu colocaria só a tipagem, ai definiria em um `beforeEach` a entrada, ou dentro de cada `it`